src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -15,6 +15,7 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
```

Comment:
`using System.Runtime.CompilerServices;`

What is this needed for?

Reply:
The `[EnumeratorCancellation]` attribute